bootstrap/container/zerotrust.go

```diff
@@ -0,0 +1,89 @@
+package container
+
+import (
+	"context"
+	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/interfaces"
+	"github.com/edgexfoundry/go-mod-bootstrap/v3/config"
+	"github.com/edgexfoundry/go-mod-core-contracts/v3/clients/logger"
+	edge_apis "github.com/openziti/sdk-golang/edge-apis"
+	"github.com/openziti/sdk-golang/ziti"
+	"net"
+	"net/http"
+	"strings"
+)
+
+const (
+	OpenZitiControllerKey = "OpenZitiController"
+	ZeroTrustKey          = "zerotrust"
+)
+
+func AuthToOpenZiti(ozController, jwt string) ziti.Context {
+	openZitiRootUrl := "https://" + ozController
+	if !strings.Contains(openZitiRootUrl, "://") {
+		openZitiRootUrl = "https://" + ozController
+	}
+	caPool, caErr := ziti.GetControllerWellKnownCaPool(openZitiRootUrl)
+	if caErr != nil {
+		panic(caErr)
+	}
+
+	credentials := edge_apis.NewJwtCredentials(jwt)
+	credentials.CaPool = caPool
+
+	cfg := &ziti.Config{
+		ZtAPI:       openZitiRootUrl + "/edge/client/v1",
+		Credentials: credentials,
+	}
+	cfg.ConfigTypes = append(cfg.ConfigTypes, "all")
+
+	ctx, ctxErr := ziti.NewContext(cfg)
+	if ctxErr != nil {
+		panic(ctxErr)
```

Comment:
i wasn't sure how to handle this. if you've asked to use security and you cannot authenticate, I opted to have the whole process panic since "nothing" will work. but i'm happy to just return an error... i'll update it and we can discuss further